pom.xml

```diff
@@ -167,6 +161,13 @@ when an elasticsearch client starts.
             <artifactId>commons-text</artifactId>
             <version>1.8</version>
         </dependency>
+        
+        <dependency>
+			<groupId>org.testcontainers</groupId>
+			<artifactId>elasticsearch</artifactId>
+			<version>1.14.2</version>
```

Comment:
I'd prefer having the TC version as a property at the beginning of the `pom.xml`. Like 

```xml
<testcontainers.version>1.14.2</testcontainers.version>
```